### ISSUE TITLE
Array of array support

### ISSUE THREAD

**Issue by arushs** [DmitryTsepelev/store_model#110]

**Comment by DmitryTsepelev** [comment]
Hi @arushs! So you want something like `{ points: [[1, 2], [3, 4]] }`, right? I guess we could add a custom type for that like we do for [dates](https://github.com/DmitryTsepelev/store_model/blob/master/docs/defining_custom_types.md).

**Comment by arushs** [comment]
Yup exactly. I made my own type for this but it seems like its a type StoreModel could support out of the box.

**Comment by DmitryTsepelev** [comment]
Maybe 🙂want to do a PR?
